lib/cocina/models/classification.rb

```diff
@@ -3,9 +3,7 @@
 module Cocina
   module Models
     class Classification < Struct
-      attribute :authority, Types::Strict::String.meta(omittable: true)
-      attribute :displayLabel, Types::Strict::String.meta(omittable: true)
-      attribute :edition, Types::Strict::String.meta(omittable: true)
+      attribute :subject, Types::Strict::Array.of(DescriptiveValue).meta(omittable: true)
```

Comment:
I don't think this is right. See https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_classification.txt